recipes-wpe/wpeframework/wpeframework-plugins_git.bb

```diff
@@ -43,9 +43,8 @@ PACKAGECONFIG ?= " \
     ${@bb.utils.contains('DISTRO_FEATURES', 'playready_nexus_svp',  'opencdmi_prnx_svp', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'playready_vg',         'opencdmi_vgrdm', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'widevine',             'opencdmi_wv', '', d)} \
-    ${@bb.utils.contains('DISTRO_FEATURES', 'wpeframework',         'network', '', d)} \
     ${@bb.utils.contains('MACHINE_FEATURES', 'wifi',                'network wifi', '', d)} \
-    compositor deviceinfo dictionary locationsync monitor remote remote-devinput spark timesync tracing ux virtualinput webkitbrowser webserver youtube \
```

Comment:
No need to disable spark Plugin

Reply:
It is needed, meta-wpe is built in different ways. And not all of those include the meta-rdk layers. For example the reference build with only meta-raspberrypi will throw an error because it doesn't know how to resolve the spark dependency if this is enabled. I suggest you add spark to the package config in the same layer that holds the spark recipes as a bbappend to the plugins.

Reply:
Hi @wouterlucas  morty spark recipes is available so we can keep it https://github.com/WebPlatformForEmbedded/meta-wpe/tree/morty/recipes-graphics/spark

Reply:
Ugh. I need more ☕️, of course it is. I don't know why I was having issues to build yesterday. I'll check it again and if it works I'll leave it in.

Reply:
Fixed in new push :)